src/backend/services/users-api/src/app/__init__.py

```diff
@@ -0,0 +1,26 @@
+import sys, os
```

Comment:
Do this file and other files also in this folder need to be in a new folder `app` inside of `users-api/src/` or can this file just go in `users-api/src`, similar to our existing services?

Reply:
Since I had to keep the database files in another folder to enable mocking for tests, I figured that it's cleaner to have the app stuff separated from the database stuff. it makes navigation easier and doesn't mix the two things.

Reply:
about the following:
> Wasn't able to make a comment on these files because the files are empty but please take a look at all of the files & folders added under services/data and `services/data/journal` and determine which ones actually need to be committed and which do not. Do we need ` src/backend/services/data/collection-0--4598222244573567696.wt` and all other similar files to be committed or can they be `.gitignore`d?
> 

> Also, do we need `src/backend/services/users-api/src/databases/__init__.py` if its empty? (Again, cant comment directly on it because its an empty file). To save writing this more, please take a look through and consider if we need the empty files within the PR and possibly removing them if we don't need them.
> 
> Also, can the other draft PR be removed now that we have this one? Seems like they are duplicates
----------------------------------

1)  I didn't bother experimenting with these files as they are automatically generated and I don't have a clue on how they're structured. Also, it didn't seem worth the effort to trim them considering they'll be removed before this code is merged to staging. They're there just for the sake of this PR.

2) We need the empty init file in every folder to initialize the folder as a package that can be imported. This is a python specific thing so it' needs to stay there.

3) just keeping the other one there until this is finalized and merged just in case I need to refer back to it for something.

Reply:
1. Ah ok well as long as these files don't stay in the PR when it merged into `staging` then all good. I would highly recommend adding them to the relevant`.gitignore` then so that you don't have to worry about deleting them before merging and you can still keep them in your local repo if you need them to run the app locally. 
2. 👍 
3. 👍